src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -91,6 +90,21 @@
 .player.ad-break,
 .player.menu-active {
   &:not(.overlay-active) {
+    :global(.playkit-video-filter) {
+      &:after {
+        position: absolute;
+        bottom: 0;
+        left: 0;
+
```

Comment:
is that a blank line? is it intentional?

Reply:
actually yes, but that's a longer discussion and i'll remove it for now